sklearn/datasets/twenty_newsgroups.py

```diff
@@ -366,10 +370,14 @@ def fetch_20newsgroups_vectorized(subset="train", remove=(), data_home=None,
         If False, raise an IOError if the data is not locally available
         instead of trying to download the data from the source site.
 
-    return_X_y : boolean, default=False.
+    return_X_y : boolean, default=False
         If True, returns ``(data.data, data.target)`` instead of a Bunch
         object.
 
+    normalize : boolean, default=True
+        If True, normalizes the returned data to unit length using
+        :func:`sklearn.preprocessing.normalize`.
+
```

Comment:
The line below (`.. versionadded:: 0.20`) should be placed under `return_X_y` parameter. You also need to add one such directive `.. versionadded:: 0.22`.